src/css/main.less

```diff
@@ -1478,3 +1472,7 @@ h3.small-caps {
     padding: 0;
   }
 }
+
+.task-details dt {
+  text-align: left;
```

Comment:
Should we stay with right align the left column until we've found a solution together with the design team?